### PR TITLE
Add latest release content for 0.26.0

### PR DESCRIPTION
**Pull request by SueChaplain**
References to full support for 0.25.0 updates, plus performance highlights.

Signed-off-by: SueChaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins website stage

**Comment by SueChaplain** [comment]
Jenkins website stage

**Comment by SueChaplain** [comment]
Jenkins website stage

File: news-page-content/latestRelease.md [hunk 1]
```diff
@@ -22,39 +22,28 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
 
-*March 2021*
+*April 2021*
 
-We're pleased to announce support for OpenJDK 16 binaries that embed OpenJ9 v0.25.0.
+We're pleased to announce the availability of Eclipse OpenJ9 v0.26.0.
 
-The OpenJDK 16 feature release contains a long list of JEPs, which include incubators and previews that might become fully supported in the next OpenJDK 17 long term support (LTS) release (September 2021).
+It's been little more than a month since our last release, which supported the OpenJDK 16
+feature release. We can confirm that the following features announced in OpenJ9 v0.25.0 are now available for OpenJDK 8 and 11:
```
Comment (pshipton) [inline news-page-content/latestRelease.md:32]: At a quick glance the reference to OpenJ9 v0.25.0 seems like a typo. Can we reword this?

i.e. ... the following features first announced in OpenJ9 v0.25.0 are now available in OpenJ9 v0.26.0 ...

**Review by pshipton — APPROVED** [review]

**Pull request merged**